app/javascript/packages/components/process-list/process-list-heading.tsx

```diff
@@ -0,0 +1,25 @@
+import { forwardRef } from 'react';
+import type { ReactNode } from 'react';
+
+interface ProcessListHeadingProps extends Record<string, any> {
```

Comment:
I don't think we need to extend here, unless we're expecting additional props beyond what's outlined in the interface.

```suggestion
interface ProcessListHeadingProps {
```

(Same goes for all other components)

Reply:
addressed in 5ec4fca83e0a9ca6ee11874362d8b462a8ed2f1f